pages/configuring_ide_idea.md

```diff
@@ -27,9 +27,9 @@ As the `node_modules/` directory is only used internally by JHipster, it can be
 
 _Note_ Some people also like to exclude the `src/main/webapp/bower_components` folder, as there is a lot of JavaScript code in that folder. However, this folder contains the frameworks and tools used when developing the application, so excluding it will cause issues with the JavaScript code support that is configured below. Therefore, it is _not recommended_ to exclude this folder.
 
-## Java Code Support
+## Spring Support (not available in Community Edition unfortunately)
```

Comment:
I'd remove unfortunately. Developers should be willing to pay for good tools IMO. ;)